### ISSUE TITLE
Attendance: mark all button issue

### ISSUE THREAD

**Issue by spewedprojects** [spewedprojects/BSP-Utility-Kt#5]
Mark all button when pressed and all individuals are marked present, it should highlight in green.
Pressing on it again undoes it's action, kind of like an undo button - if a few employees were marked present, and somebody mistakenly presses mark all, somewhere to undo that mistake should be there, retaining the previously few marked employees.
